example/App.js

```diff
@@ -127,6 +127,7 @@ export default class App extends Component {
   getSyncDeviceInfo() {
     let deviceJSON = {};
 
+    deviceJSON.uniqueId = getUniqueId();
```

Comment:
This is the synchronous API exercise, so should be the sync version? May need the import above

```suggestion
    deviceJSON.uniqueId = getUniqueIdSync();
```

...then I would expect the getUniqueId() call to be down below, somewhere in the block that starts near line 200 ?

Reply:
I've updated examples for sync and async device info.